NewTsRebase/src/mpegTsPacketizer.c

```diff
@@ -0,0 +1,217 @@
+#include <stdlib.h>
+#include <string.h>
+#include "mpegTsPacketizer.h"
+#include "mpegTs.h"
+#include "constants.h"
+#include "id3v2lib.h"
+
+
+ID3v2_struct* parse_json(char * json){
+    const char * regex_objectType="\"objectType\":\"(.*)\",";
+    const char * regex_timestamp="\"timestamp\":(.*)}";
+    const char * find_text=json;
+    ID3v2_struct* ID3v2_struct_t=new_ID3v2_struct();
+    if (ID3v2_struct_t==NULL){
+        puts("failed to allocate memory for ID3v2 struct.\n");
+        return NULL;
+    }
+    ID3v2_struct_t->objectType=match_regex(regex_objectType, find_text);
+    char *UnixTimeStamp_string=match_regex(regex_timestamp, find_text);
+   ID3v2_struct_t->UnixTimeStamp = (long)strtold(UnixTimeStamp_string,NULL);
```

Comment:
check for UnixTimeStamp_string!=null before parsing it
+ check if parsing was sucessfull  + json parsing